### PR TITLE
Supply Chain Pre-Release Merge into Master

### PR DESCRIPTION
**Pull request by wsobel**
* Pre Release Version of SCRO for review.
*  Vote has passed, now merging into master

### CODE REVIEW

File: supplychain/SupplyChainReferenceOntology.rdf [hunk 1]
```diff
@@ -12,63 +13,68 @@
...
-		<dcterms:contributor xml:lang="en-US">Boonserm Kulvatanyou, NIST</dcterms:contributor>
-		<dcterms:contributor xml:lang="en-US">Chris Will, National Center for Ontological Research (NCOR)</dcterms:contributor>
-		<dcterms:contributor xml:lang="en-US">Dusan Sormaz, Ohio University</dcterms:contributor>
-		<dcterms:contributor xml:lang="en-US">Evan Wallace, NIST</dcterms:contributor>
+		<rdfs:label xml:lang="en-us">Supply Chain Ontology</rdfs:label>
+		<dcterms:abstract xml:lang="en-us">The IOF Supply Chain Ontology aims to represent the generic constructs (including classes and properties) related to the domain of supply chain and logistics. The Supply Chain Ontology uses BFO as the top-level ontology and IOF Core as the mid-level ontology. The purpose of the ontology is to serve as a foundation for ensuring consistency and interoperability across various supply chain and logistics ontologies that use IOF reference ontolgies.</dcterms:abstract>
+		<dcterms:contributor xml:lang="en-us">Boonserm Kulvatanyou, NIST</dcterms:contributor>
```
Comment (mereolog) [inline supplychain/SupplyChainReferenceOntology.rdf:32]: I think that https://oagiscore.atlassian.net/wiki/spaces/IOF/pages/4532142081/IOF+Annotation+Property+Guide+V2.2#Summary-of-Annotation-Requirements requires 'en-US'.

This issue should have been found by the hygiene tests, but it was not.
https://github.com/iofoundry/ontology/pull/47 should fix this bug in hygiene tests.

**Review by mereolog — CHANGES_REQUESTED** [review]
There is a couple of ill-formed axioms/definition: [iof_supplychain_illformed_axioms_faae427.xlsx](https://github.com/iofoundry/ontology/files/10782367/iof_supplychain_illformed_axioms_faae427.xlsx)
See also https://github.com/iofoundry/ontology/issues/46#issuecomment-1434757440.

**Review by mereolog — APPROVED** [review]

**Review by ElisaKendall — APPROVED** [review]
I'll approve this but I thought we were renaming the ontology from SupplyChainReferenceOntology to SupplyChain?

Otherwise, really great job on eliminating all the hygiene issues!

**Comment by fameri** [comment]
> I'll approve this but I thought we were renaming the ontology from SupplyChainReferenceOntology to SupplyChain?
> 
> Otherwise, really great job on eliminating all the hygiene issues!

I renamed SupplyChainReferenceOntology.rdf to SupplyChain.rdf (and updated the IRIs) in the branch. Should I submit another pull request? 
Farhad

**Review by mereolog — CHANGES_REQUESTED** [review]
@fameri after the rename of the ontology you need to update all references to the previous file name and the previous IRI in the catalog and metadata files. 

I can do it for you if you want.

**Review by MilosDrobnjakovic — APPROVED** [review]

**Comment by mereolog** [comment]
> > @fameri after the rename of the ontology you need to update all references to the previous file name and the previous IRI in the catalog and metadata files.
> > I can do it for you if you want.
> 
> @mereolog I actually updated the IRIs as well. Can you please check and pull?

There are still 7 references to [SupplyChainReferenceOntology](https://spec.industrialontologies.org/ontology/supplychain/SupplyChainReferenceOntology/) - see below.

image

We need to rename at least those in catalog and metadata files.

**Review by mereolog — CHANGES_REQUESTED** [review]
There are still two references to SupplyChainReferenceOntology in https://github.com/iofoundry/ontology/blob/bdcbcd066ff6d20f530e75ce7188ec9dd9bb7436/supplychain/Metadatasupplychain.rdf.

**Review by mereolog — APPROVED** [review]

**Pull request merged**